src/lib/kvbCheckVc.ts

```diff
@@ -75,6 +76,10 @@ export function buildKbvCheckArtifacts(
   const metadata = buildThing(createThing({ url: metadataUri }))
     .addUrl(RDF.type, GOV_UK_CREDENTIAL)
     .addUrl(GOV_UK_HAS_CREDENTIAL, fileUri)
+    .addStringEnglish(
+      GOV_UK_VC_DESCRIPTION,
```

Comment:
It'd be interesting to talk to Pat about if there are any 'standard' properties we could use for these instead. For example, I this one could be [`rdfs:label` ](https://www.w3.org/TR/rdf-schema/#ch_label)